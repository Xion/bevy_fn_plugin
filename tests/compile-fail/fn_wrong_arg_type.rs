#[macro_use] extern crate bevy_fn_plugin;

#[bevy_plugin]
fn fn_weird_arg(_: usize) {}

fn main() {}
