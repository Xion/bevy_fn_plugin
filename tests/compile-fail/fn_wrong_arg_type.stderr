error: Invalid plugin function signature
 --> tests/compile-fail/fn_wrong_arg_type.rs:4:1
  |
4 | fn fn_weird_arg(_: usize) {}
  | ^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: Function should take a single argument of the type `&mut bevy::app::App`
